vue/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "vue",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vue-tsc && vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@rango-dev/widget-embedded": "^0.25.0",
+    "react": "^18.2.0",
```

Comment:
Can you explain why 'react' and 'react-dom' should still be included in 'dependencies'?

Reply:
Because the new package we use (`veaury`) does not have `react` & `react-dom` itself and only has them as peer dependencies. As a result, they need to be installed as well, like in this example:
https://github.com/devilwjp/veaury/blob/master/dev-project-vue3/package.json